sonic-xcvrd/xcvrd/xcvrd_utilities/y_cable_helper.py

```diff
@@ -897,26 +889,26 @@ def post_port_mux_info_to_db(logical_port_name, table):
-                 ('run_slot2_tor1', str(mux_info_dict["run_slot2_tor1"])),
-                 ('commit_slot1_tor1', str(mux_info_dict["commit_slot1_tor1"])),
-                 ('commit_slot2_tor1', str(mux_info_dict["commit_slot2_tor1"])),
-                 ('empty_slot1_tor1', str(mux_info_dict["empty_slot1_tor1"])),
-                 ('empty_slot2_tor1', str(mux_info_dict["empty_slot2_tor1"])),
-                 ('build_slot1_tor2', str(mux_info_dict["build_slot1_tor2"])),
-                 ('build_slot2_tor2', str(mux_info_dict["build_slot2_tor2"])),
-                 ('version_slot1_tor2', str(mux_info_dict["version_slot1_tor2"])),
-                 ('version_slot2_tor2', str(mux_info_dict["version_slot2_tor2"])),
-                 ('run_slot1_tor2', str(mux_info_dict["run_slot1_tor2"])),
-                 ('run_slot2_tor2', str(mux_info_dict["run_slot2_tor2"])),
-                 ('commit_slot1_tor2', str(mux_info_dict["commit_slot1_tor2"])),
-                 ('commit_slot2_tor2', str(mux_info_dict["commit_slot2_tor2"])),
-                 ('empty_slot1_tor2', str(mux_info_dict["empty_slot1_tor2"])),
-                 ('empty_slot2_tor2', str(mux_info_dict["empty_slot2_tor2"]))
+                 ('build_slot1_tor_self', str(mux_info_dict["build_slot1_tor_self"])),
+                 ('build_slot2_tor_self', str(mux_info_dict["build_slot2_tor_self"])),
+                 ('version_slot1_tor_self', str(mux_info_dict["version_slot1_tor_self"])),
+                 ('version_slot2_tor_self', str(mux_info_dict["version_slot2_tor_self"])),
+                 ('run_slot1_tor_self', str(mux_info_dict["run_slot1_tor_self"])),
+                 ('run_slot2_tor_self', str(mux_info_dict["run_slot2_tor_self"])),
+                 ('commit_slot1_tor_self', str(mux_info_dict["commit_slot1_tor_self"])),
+                 ('commit_slot2_tor_self', str(mux_info_dict["commit_slot2_tor_self"])),
+                 ('empty_slot1_tor_self', str(mux_info_dict["empty_slot1_tor_self"])),
+                 ('empty_slot2_tor_self', str(mux_info_dict["empty_slot2_tor_self"])),
+                 ('build_slot1_tor_peer', str(mux_info_dict["build_slot1_tor_peer"])),
+                 ('build_slot2_tor_peer', str(mux_info_dict["build_slot2_tor_peer"])),
+                 ('version_slot1_tor_peer', str(mux_info_dict["version_slot1_tor_peer"])),
+                 ('version_slot2_tor_peer', str(mux_info_dict["version_slot2_tor_peer"])),
+                 ('run_slot1_tor_peer', str(mux_info_dict["run_slot1_tor_peer"])),
+                 ('run_slot2_tori_peer', str(mux_info_dict["run_slot2_tor_peer"])),
+                 ('commit_slot1_tor_peer', str(mux_info_dict["commit_slot1_tor_peer"])),
+                 ('commit_slot2_tor_peer', str(mux_info_dict["commit_slot2_tor_peer"])),
+                 ('empty_slot1_tor_peer', str(mux_info_dict["empty_slot1_tor_peer"])),
+                 ('empty_slot2_tor_peer', str(mux_info_dict["empty_slot2_tor_peer"]))
                  ])
```

Comment:
I believe you're adding all elements from `mux_info_dict` into the DB and using the key names from the dict as the key names for the DB. Therefore, you should be able to simplify this to one line:

```python
fvs = swsscommon.FieldValuePairs(list(mux_info_dict.items()))
```

Here, `list(mux_info_dict.items())` creates a list of tuples. However, I notice you're converting a lot of the values to strings. If this is necessary, then you might need to create the list of tuples then walk the list and convert the second element of each tuple to a string, but it would be a lot less code and maintenance than this, as long as we intend to push every item from `mux_info_dict` to the DB.

We can save this for a future cleanup PR.